07/hangman/hangman_game.py

```diff
@@ -0,0 +1,33 @@
+from hangman_settings import vytvor_retezec
+from hangman_pismeno import pismeno
+from hangman_score import score
+from hangman_pismeno_ve_slove import dosazeni_pismene
+from random import choice
+
+
+def hangman_game(seznam):
+    pokus = 0
+    print()
+    print('Hra: OBĚŠENEC')
+    print('Uhádni písmeno ve slově a poskládej tak slovo')
+    slovo = choice(seznam)
+    retezec = vytvor_retezec(slovo)
+    print(retezec)
+    print(slovo)
+
+    while True:
+        pismenko_k_dosazeni = pismeno()
+        retezec = dosazeni_pismene(slovo, pismenko_k_dosazeni, retezec)
+        print()
+        if pismenko_k_dosazeni not in slovo:
+            pokus += 1
+        print(score(retezec, pismenko_k_dosazeni, pokus, slovo))
+        if "_" not in retezec:
+            break
+        elif pokus == 10:
+            quit()
```

Comment:
Ten `quit()` by si si naozaj nemala zvykať používať. Používať by sa mal len v Python konzoli, v produkčnom kóde je to vyslovene zakázané. Takže za mňa, ak to bude možné, tak ho prosím v kóde nepoužívaj.

Viac info na prečítanie napr tu: 
https://stackoverflow.com/questions/19747371/python-exit-commands-why-so-many-and-when-should-each-be-used#:~:text=After%20all%2C%20one%20of%20the,be%20used%20in%20the%20interpreter

Ako som písala v komentári vyššie, toto ide prepísať priamo do while podmienky. Ak by si trvala na tom mať to tu, tak rovnako ako na r.26 tu má byť break.